src/components/NavigationButtons.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { Button, SecondaryButton, screenBreakpoints } from './Common'
+import { I } from './Typography'
+
+const StyledButton = styled(Button)`
+  margin: 0;
+
+  @media (max-width: ${screenBreakpoints.xs}px) {
+    margin-top: 0.5em;
+  }
+`
+
+const StyledSecondaryButton = styled(SecondaryButton)`
+  margin: 0;
+`
+
```

Comment:
```suggestion
```

Reply:
Is this suggestion to remove the blank line?

Reply:
Oh it's to remove `StyledSecondaryButton` and the newline after since it's not needed anymore

Reply:
OH wait I am dumb sorry, brain fart 😭